src/core/display/TransformStatic.js

```diff
@@ -1,4 +1,4 @@
-import math from '../math';
+import * as math from '../math';
```

Comment:
Yeah we need to do a pass where we change all the `import * as` into only the imports needed. Just didn't want to hold up this entire PR for that pass.